concent_api/core/subtask_helpers.py

```diff
@@ -82,42 +98,41 @@ def finalize_claim_for_acceptance_case() -> None:
             finalize_claim_for_acceptance_case,
             using='control',
         )
-
+    elif subtask.state == Subtask.SubtaskState.VERIFICATION_FILE_TRANSFER.name:  # pylint: disable=no-member
         update_subtask_state(
             subtask=subtask,
-            state=Subtask.SubtaskState.ACCEPTED.name,  # pylint: disable=no-member
```

Comment:
It looks like lot of important things would be changed. It is not, if somebody will review this please use PyCharm to see pull request. Github did it very mistakable, in PyCharm pull request you can see it much better

Reply:
Yes I agree, when I was looking on it in Github I wasn't sure if I am looking at the correct pull request :)

Reply:
That's what happens when you move around a lot of very similar blocks of code. Neither github nor PyCharm is actually right or wrong here. This change can be done both by swapping the whole blocks (as Pycharm probably tells you) or by swapping just the values github shows. They can't know which way you actually did it. That would require understanding the meaning of this code. There are many situations where you'll see the opposite: you changed only a few values but the diff tells you that whole blocks were moved.